app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -23,8 +23,7 @@ interface FooterProps {
 const Header = ({ formattedEndTime, formattedStartTime, title, isPast }: HeaderProps) => (
   <View style={$headerContainer}>
     <Text style={isPast ? $pastTimeText : $timeText}>
-      {formattedStartTime}
-      {!!formattedEndTime && ` - ${formattedEndTime}`}
+      {`${formattedStartTime}${formattedEndTime ? ` - ${formattedEndTime}` : ""} PT`}
```

Comment:
I'm still thinking about that `formattedTime` function. I'll see if I can get to that today.

Reply:
fixed this here: https://github.com/infinitered/ChainReactApp2023/pull/149/commits/f6fc2d5469b62c49985db6c5bc7b561a056b11f7